common/network-common/src/test/java/org/apache/spark/network/ChunkFetchIntegrationSuite.java

```diff
@@ -143,37 +143,38 @@ public void releaseBuffers() {
   }
 
   private FetchResult fetchChunks(List<Integer> chunkIndices) throws Exception {
-    TransportClient client = clientFactory.createClient(TestUtils.getLocalHost(), server.getPort());
-    final Semaphore sem = new Semaphore(0);
-
     final FetchResult res = new FetchResult();
```

Comment:
nit: Can we put this line into `try` since this line was originally executed after `clientFactory.createClient`.

Reply:
No we can't, since the `res` is returned by the function :-)

Reply:
Any time